server/datastore/mysql/migrations/tables/20230706141219_QueriesTableSchemaChanges.go

```diff
@@ -0,0 +1,49 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230706141219, Down_20230706141219)
+}
+
+func Up_20230706141219(tx *sql.Tx) error {
+	// If we want to drop the uniq constraint on queries.name, we first need to remove this
+	// constraint on scheduled_queries, due to a FK constraint scheduled_queries (query_name) =>
+	// queries (name).
+	if _, err := tx.Exec(`
+		ALTER TABLE scheduled_queries DROP FOREIGN KEY scheduled_queries_query_name;
+	`); err != nil {
+		return errors.Wrap(err, "removing FK on scheduled_queries")
+	}
+
+	if _, err := tx.Exec(`
+		ALTER TABLE queries
+			DROP INDEX idx_query_unique_name,
+			DROP INDEX constraint_query_name_unique,
+
+			ADD team_id INT(10) UNSIGNED DEFAULT NULL,
+			ADD team_id_char CHAR(10) DEFAULT '',
+
+			ADD platform VARCHAR(255) DEFAULT NULL,
```

Comment:
Let's make it `DEFAULT ''` (https://github.com/fleetdm/fleet/pull/11742/files shows platform being returned as empty string in the API)

Reply:
Done in https://github.com/fleetdm/fleet/pull/12673/commits/2b8dd6571685f9040f4702deb44b45bbbdea78b8